source/sass/views/initiatives.scss

```diff
@@ -5,3 +5,62 @@
 .bottom-line {
   border-bottom: 1px solid $light-gray;
 }
+
+// Fellowships--Support
+
+.fellow-support {
+  &-intro {
+    &-help {
+      font-weight: 600;
+    }
+  }
+
+  &-optim {
+    max-width: 45%;
+
+    @media (min-width: $bp-md) {
+      max-width: 85%;
+    }
+  }
+
+  &-logo {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/box-align) it seems like we should avoid using `box-align`.

Reply:
For most of the flexbox related rules we can probably replace them by applying corresponding Bootstrap classes. See https://getbootstrap.com/docs/4.0/utilities/flex/

Reply:
oh padding/margin as well

https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
Ideally we want to avoid heavily nested classes but I think this issue will be gone once the rules are replaced by Bootstrap classes.

Reply:
:+1: